src/views/download-view.js

```diff
@@ -162,21 +164,29 @@ class DownloadView extends PolymerElement {
                     <!--<code>ff-use-class</code>, -->
                 </p>
             </div>
+            
+            <template is="dom-if" if="[[isSelectedVersion3XX]]">
+                <div class="row">
+                    <p style="padding: 22px; border: 4px solid grey; background-color: lightsteelblue">
+                        Currently only a full download is supported for <b>v.{{selectedVersion}}</b>. Please contact us if you think you definitely need a reduced bundle.
```

Comment:
I'd rather go with a slightly more formal wording.
```suggestion
                        At the moment, only a full download is supported for <b>v.{{selectedVersion}}</b>. Please contact us if you must use a reduced bundle.
```